frontend/src/views/NewStore/NewStore.js

```diff
@@ -10,6 +10,8 @@ import BusinessForm from './BusinessForm';
 import WaitingForVerify from './WaitingForVerify';
 import { useUserContext } from 'context/UserContext';
 import { useTranslation } from 'react-i18next';
+import { signInWithGoogle } from "../../shared/firebase.utils";
```

Comment:
I would like if you could make this a dynamic import, so that we dont have to load all that for all the clients. https://reactjs.org/docs/code-splitting.html